### PR TITLE
[KOGITO-2007] - Prepare Kogito Operator for 0.11.x version iteration

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

See: https://issues.redhat.com/browse/KOGITO-2007

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: deploy/olm-catalog/kogito-operator/0.11.0/kogito-operator.v0.11.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+  - cloud
+  - kogito
+  - quarkus
+  - springboot
+  - operator
+  - native
+  - automation
+  labels:
+    alm-owner-kogito: kogito-operator
+    operated-by: kogito-operator.0.10.0
```
Comment (ricardozanini) [inline deploy/olm-catalog/kogito-operator/0.11.0/kogito-operator.v0.11.0.clusterserviceversion.yaml:610]:
```suggestion
    operated-by: kogito-operator.0.11.0
```

File: deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
   name: alpha
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
```
Comment (ricardozanini) [inline deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml:4]:
```suggestion
- currentCSV: kogito-operator.v0.9.1
```
We are not going to update the dev-preview channel yet

File: deploy/olm-catalog/kogito-operator/0.11.0/kogito-operator.v0.11.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,668 @@
...
+          },
+          "spec": {
+            "replicas": 1
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.10.0
```
Comment (ricardozanini) [inline deploy/olm-catalog/kogito-operator/0.11.0/kogito-operator.v0.11.0.clusterserviceversion.yaml:75]:
```suggestion
    containerImage: quay.io/kiegroup/kogito-cloud-operator:0.11.0
```

File: test/.default_config [hunk 4]
```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=0.10.0-rc1
-tests.services-image-version=0.10.0-rc1
-tests.examples-ref=master
```
Comment (sutaakar) [inline test/.default_config:3]: the examples reference should stay pointing to master

File: test/.default_config [hunk 5]
```diff
@@ -1,3 +1,3 @@
-tests.build-image-version=0.10.0-rc1
```
Comment (sutaakar) [inline test/.default_config:1]: This should IMHO point to latest available image tags, it is 0.10.0-rc2 currently.

Comment (sutaakar) [reply test/.default_config:1]: @radtriste Wouldn't it have sense to point to `latest` tag?

**Comment by radtriste** [comment]
Do we have a PR already for releasing Kogito Operator 0.10.0 ?...

File: test/.default_config (hunk 5 above)
Comment (radtriste) [reply test/.default_config:1]: in master it would

File: test/.default_config (hunk 4 above)
Comment (radtriste) [reply test/.default_config:3]: yep, the bump-version is guessing to have 0.11.x branch already. will update the bump-version script

Comment (radtriste) [reply test/.default_config:3]: https://github.com/kiegroup/kogito-cloud-operator/pull/321

File: deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml [hunk 6]
```diff
@@ -1,7 +1,7 @@
 channels:
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.11.0
   name: alpha
-- currentCSV: kogito-operator.v0.10.0
+- currentCSV: kogito-operator.v0.9.1
```
Comment (radtriste) [inline deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml:4]: shouldn't we have here 0.10.0 or 0.11.0 ?

Comment (sutaakar) [reply deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml:4]: no, that is a dev-preview channel, it should point to DP version (0.9.1)

Comment (radtriste) [reply deploy/olm-catalog/kogito-operator/kogito-operator.package.yaml:4]: oki :)

**Comment by ricardozanini** [comment]
> Do we have a PR already for releasing Kogito Operator 0.10.0 ?...

I don't think so, I believe @spolti is working on it as we speak :D

**Comment by ricardozanini** [comment]
@spolti please let's merge this ASAP, since my work on [KOGITO-128](https://issues.redhat.com/browse/KOGITO-128) will mess around everything :p

**Comment by spolti** [comment]
@radtriste @sutaakar can you please look again the default_config file?

**Pull request merged**